code/model/editableformfields/EditableFormField.php

```diff
@@ -984,4 +1003,62 @@ public function EffectiveDisplayRules()
         }
         return $this->DisplayRules();
     }
+
+	public function FormatDisplayRules()
```

Comment:
Same deal - if this isn't required by the template can it please be named with camelCase?